Mention priority 0 and 2 (#284)
bskinn/sphobjinv: doc/source/syntax.rst

``{priority}``
    Flag for `placement in search results
    <https://github.com/sphinx-doc/sphinx/blob/2f60b44999d7e610d932529784f082fc1c6af989/sphinx/domains/__init__.py#L370-L381>`__. Most will be ``1`` (standard priority) or

>     ``-1`` (omit from results) for documentation built by Sphinx. ``0`` means higher priority, and ``2`` means lower priority.

Reviewer: Seems like a good idea. I'd word it differently:

```suggestion
    ``-1`` (omit from results) for documentation built by Sphinx;
    values of ``0`` (higher priority) or ``2`` (lower priority) may also occur.
```
